Code/GraphMol/FileParsers/SDWriter.cpp

```diff
@@ -114,16 +114,16 @@ void _MolToSDStream(std::ostream *dp_ostream, const ROMol &mol, int confId,
     // if use did not specify any properties, write all non computed properties
     // out to the file
     STR_VECT properties = mol.getPropList();
-    STR_VECT compLst;
-    mol.getPropIfPresent(detail::computedPropName, compLst);
-
+    STR_VECT compLst = mol.getComputedPropList();
+    
     STR_VECT_CI pi;
     for (pi = properties.begin(); pi != properties.end(); pi++) {
       // ignore any of the following properties
       if (((*pi) == detail::computedPropName) ||
-          ((*pi) == common_properties::_Name) || ((*pi) == "_MolFileInfo") ||
-          ((*pi) == "_MolFileComments") ||
-          ((*pi) == common_properties::_MolFileChiralFlag)) {
+          ((*pi) == common_properties::getPropName(common_properties::_Name)) ||
+          ((*pi) == common_properties::getPropName(common_properties::MolFileInfo)) ||
+          ((*pi) == common_properties::getPropName(common_properties::MolFileComments)) ||
+          ((*pi) == common_properties::getPropName(common_properties::_MolFileChiralFlag))) {
         continue;
```

Comment:
A nice micro-optimization possible here by doing these comparisons as integers instead of strings. Should speed up writing SDF files with lots of SD data. 

Is it possible properties and compLst are nearly identical long vectors of the same strings? If so, the std::find below will also be O(N^2) in this worst case scenario.